tdvt/tdvt/exprtests/initialsql/setup.initialsql.notemptable.int.txt

```diff
@@ -0,0 +1 @@
+RAWSQLAGG_STR("SUM(%1)", [int0])
```

Comment:
What is actually being tested with this test case?  The initial sql set isn't referenced by the test, like it is [here](https://github.com/tableau/connector-plugin-sdk/blob/master/tdvt/tdvt/exprtests/initialsql/setup.initialsql.temptable.txt)

Reply:
Yes I had same concern. I had tried to add better sql statement in the calcs but we have different issues with individual connections. I will pull back these changes as this is getting covered by other test cases.